Epam.TestAutomation.Tests/CareersPageTests.cs

```diff
@@ -0,0 +1,37 @@
+using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Core.Utils;
+using Epam.TestAutomation.Web.PageObjects.Pages;
+using NUnit.Framework;
+
+namespace Epam.TestAutomation.Tests;
+[TestFixture] [Parallelizable(ParallelScope.All)]
+public class CareersPageTests : BaseTest
+{
+    private CareersPage careersPage;
+
+    [SetUp]
+    public void SetUp()
+    {
+        careersPage = new CareersPage();
```

Comment:
all steps in the test should start from MainPage. You click on Careers button and find button on the main page and then make verification on the careers page.